pom.xml

```diff
@@ -62,6 +72,15 @@
             <name>m2-dv8tion</name>
             <url>https://m2.dv8tion.net/releases</url>
         </repository>
+        <repository>
+            <id>central</id>
+            <name>bintray</name>
+            <url>https://jcenter.bintray.com</url>
+        </repository>
```

Comment:
What is JCenter used for? And why is it identified as central?

```suggestion
```

Reply:
Oops, not sure why this was added. Builds fine without it so can be removed